file.txt

```diff
@@ -1,3 +1,4 @@
 Git Branch Demo
 main branch change
-feature-1 change
\ No newline at end of file
+feature-1 change
+add change tru PR
```

Comment:
Change tru to through

Reply:
Fixed in commit [03b388baa0b3350eadc95855cdbfdf43b7dd9b04]

Reply:
Fixed in commit 03b388baa0b3350eadc95855cdbfdf43b7dd9b04